Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -184,6 +190,58 @@ int main(int argc, char** argv)
         app->quit();
     }));
 
+    auto& view_menu = menubar->add_menu("&View");
+    view_menu.add_action(GUI::CommonActions::make_fullscreen_action([&](auto&) {
+        window->set_fullscreen(!window->is_fullscreen());
+    }));
+
+    auto& rotation_axis_menu = view_menu.add_submenu("Rotation Axis");
+    auto rotation_x_action = GUI::Action::create_checkable("x", [&widget]() {
+        widget.rotate_x = !widget.rotate_x;
+    });
+    auto rotation_y_action = GUI::Action::create_checkable("y", [&widget]() {
+        widget.rotate_y = !widget.rotate_y;
+    });
+    auto rotation_z_action = GUI::Action::create_checkable("z", [&widget]() {
+        widget.rotate_z = !widget.rotate_z;
+    });
+
+    rotation_axis_menu.add_action(*rotation_x_action);
+    rotation_axis_menu.add_action(*rotation_y_action);
+    rotation_axis_menu.add_action(*rotation_z_action);
+
+    rotation_x_action->set_checked(true);
+    rotation_z_action->set_checked(true);
+
+    auto& rotation_speed_menu = view_menu.add_submenu("Rotation Speed");
+    GUI::ActionGroup rotation_speed_actions;
+    rotation_speed_actions.set_exclusive(true);
+
+    auto no_rotation_action = GUI::Action::create_checkable("No Rotation", [&]() {
```

Comment:
```suggestion
    auto no_rotation_action = GUI::Action::create_checkable("N&o Rotation", [&] {
```

Reply:
I 100% agree with adding keyboard shortcuts for everything, but is there a reason that you prefer `N&o Rotation` over `&No Rotation`? Is `N` usually used for something else?

Reply:
Oh wait, nevermind about this comment. I see why now :)